ios/README.md

```diff
@@ -74,6 +76,23 @@ the result with a `UIAlertViewController` and logs the result to the console usi
 
 ### Integration
 
+#### Cocoapods
+
+To integrate [`LinkKit.framework`][linkkit] into your application using
+[CocoaPods](https://cocoapods.org/#install) add the next line to your Podfile:
+
+```
+pod 'Plaid'
+```
+
+and run the following command:
+`pod install`
+
+To update [`LinkKit.framework`][linkkit] in the future run:
+`pod update Plaid`
```

Comment:
Yes, otherwise the binary will fail App Store upload validation due to the executable containing unsupported architectures; namely i386 and x86_64 needed for iOS Simulator builds.